test/functions/ferguson/query/owned-inner2.chpl

```diff
@@ -0,0 +1,5 @@
+proc f(arg:owned(?t)) {
```

Comment:
I would not have expected this pattern to be desirable. Does ``owned`` width parentheses come up anywhere else?

Reply:
Part of the reason I made this .future is that this pattern worked before this PR. It appeared as `_owned(?t)` but the parser doesn't actually distinguish between `_owned` and `owned` in this case. If you don't think we should support `owned(?t)` long term as a synonym for `owned ?t`, please comment on #14431.